packages/dd-trace/src/appsec/index.js

```diff
@@ -1,5 +1,6 @@
 'use strict'
```

Comment:
we need to advertise the new [capability](https://datadoghq.atlassian.net/wiki/spaces/RC/pages/2594702792/Client+capabilities+list#Capabilities) `ASM_REQUEST_BLOCKING` when appsec starts now.
Something like
`rc.updateCapabilities(Capabilities.ASM_REQUEST_BLOCKING, true)` when enabling appsec and
`rc.updateCapabilities(Capabilities.ASM_REQUEST_BLOCKING, false)` when disabling appsec.
Need to add the capability in remote_config/capabilities.js of course.